pilot/pkg/model/push_context.go

```diff
@@ -115,12 +115,15 @@ type processedDestRules struct {
 type processedAuthnPolicies struct {
 	policies map[host.Name][]*authnPolicyByPort
 	// default cluster-scoped (global) policy to be used if no other authn policy is found
-	defaultMeshPolicy *authn.Policy
+	defaultMeshPolicy     *authn.Policy
+	defaultMeshPolicyMeta ConfigMeta
```

Comment:
do you want to store by value?  fields above are using pointer and i peek a bit, `ConfigMeta` contains lots of fields/maps, etc.

Reply:
Done. How golang handle object and pointer is a black magic :)